ui/scss/component/_button.scss

```diff
@@ -118,6 +118,26 @@
   }
 }
 
+.vjs-button--loop.vjs-button {
```

Comment:
Instead of duplicating these styles, just add this class to the rule above
```
.vjs-button--theater-mode,
.vjs-button--loop {
  ...
  ```

Reply:
thanks, just kept a separate one for the different icons